packages/dialtone-vue2/components/rich_text_editor/extensions/suggestion/SuggestionList.vue

```diff
@@ -70,6 +70,7 @@ export default {
 
   methods: {
     onKeyDown ({ event }) {
+      if (this.items.length === 0) return;
```

Comment:
This seems like not working, I can still trigger errors in console pressing arrow keys

Reply:
> This seems like not working, I can still trigger errors in console pressing arrow keys

hmm can you tell me the steps you take to trigger this?

Reply:
ahh, seems like `this.items` is not empty on a second search so it always pass this validation.

Check the video, after I type `@` I start to press the down arrow key.
Then I delete it and type again `@` and start to press down arrow key, console error appears.

https://github.com/dialpad/dialtone/assets/89984179/caf21634-7675-42e2-b51f-08dfce178343

Reply:
There is another issue but it was there before.
If you press up arrow key it triggers a console error that seems to be related to destroy the popup (probably because you are moving out the @)

To reproduce type `@` and quickly up arrow key.

https://github.com/dialpad/dialtone/assets/89984179/14bf055b-d0a1-4590-9da2-c87c5e2b93eb

Reply:
> Check the video, after I type `@` I start to press the down arrow key.
> Then I delete it and type again `@` and start to press down arrow key, console error appears.

Ah yeah, seems to happen if you press the arrow key before the dialog is loaded

Reply:
The variables for the component and the popup in suggestion.js were not getting set to null after being destroyed. They still had an object in them even though they were destroyed which kept many things "alive" when they shouldn't have been.

Should be fixed now.